### ISSUE TITLE
Incorrect error message when creating a future pair request

### ISSUE THREAD

**Issue by RocDavenport** [agency-of-learning/PairApp#192]
**Describe the bug**
When attempting to create a Pair Request dated beyond 1 month in the future, the error message states "Error! Please fix inputs When field must not be in the past"

**To Reproduce**
1. Go to 'Pair Requests'
2. Select an 'Invitee'
3. Enter a 'Date' more than a month in the future.
4. Select a 'Duration'
5. Select an 'Create Pair Request
6. See error

**Expected behavior**
I expect to see an error message that is applicable to the date selected.

**Screenshots**
<img width="1277" alt="AOL_my_request_errorpng" src="https://github.com/agency-of-learning/PairApp/assets/134003653/c2db42ce-448f-4ed8-b0e1-cc5941621897">

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser Chrome 
 - Version 114

**Additional context**

**Comment by GALTdea** [comment]
I think you should tackle this one @RocDavenport.

**Comment by afogel** [comment]
I agree that the error message could probably be improved, but might this be an issue with the fact that the invitee isn't selected there?

**Comment by RocDavenport** [comment]
No I think what happened here is once the error was thrown, the invitee
field was defaulted back to no one. The screenshot was captured after the
failed attempt to schedule a pair request.

**Comment by RocDavenport** [comment]
And so it is! Thank you!

**Comment by afogel** [comment]
@RocDavenport wait, does that mean that the cause was that invitee was unselected?

**Comment by RocDavenport** [comment]
@Ariel no, and I apologize for the confusion.

When I attempted to schedule the Pair Request, I received the error
message. At the time the attempt was made, all fields were completed.

I believe the error stems from the fact that I accidentally entered a date
in September (more than 1 month away).

When I then took the screenshot, it appeared as if no invitee was selected
at the time of the pair request attempt, but that is not the case.
